src/utils/fake-backend.js

```diff
@@ -1,4 +1,19 @@
 // array in local storage for registered users
+
+let usersArray = [
+  {
+    firstName: 'ahmed',
+    lastName: 'aly',
+    username: 'ahmed.aly',
+    password: 'umbrage',
+    id: 1,
+  },
+];
+
+let users_serialized = JSON.stringify(usersArray);
+
+localStorage.setItem('users', users_serialized);
+
```

Comment:
Im not sure if this is right but this is the only way I could get past the login screen.

Reply:
let's hop on a call to discuss